.gitlab-ci.yml

```diff
@@ -1,7 +1,9 @@
-# the following block is needed for the shared Docker-based runner
+# The following block is needed for the shared Docker-based runner
+# For local runners you might want to enable the overlay driver:
+# https://docs.gitlab.com/ce/ci/docker/using_docker_build.html#using-the-overlayfs-driver
+
 image: docker:git # docker and git clients
 variables:
-  DOCKER_DRIVER: "overlay" # recommended for docker-in-docker, needs overlay kernel module loaded on host
```

Comment:
I've taken a look at the gitlab.com website for a few minutes, but it's not yet clear to me (i.e. more than an average user) what I should do for the shared runner on my Gitlab CI (i.e. DOCKER_DRIVER: "overlay" if I'm not mistaken). Why don't we keep this line, comment it out, and mention that this line is necessary for the shared runner (or anything appropriate)?

Reply:
Setting ` DOCKER_DRIVER: "overlay"` (or "overlay2"?) is not enough.
If you don't do the other step like configuring your kernel, you will end up with the problem described in #213.
However, If you set-up you runner you can even enable the overlay driver [for all jobs](https://docs.gitlab.com/ce/ci/docker/using_docker_build.html#use-driver-for-every-project), so you don't have to include this in each yaml file.

You do not need any of this and can just got with the default, but it might be slower..
According to my test you don't save anything if you set it for the shared runners explicitly.

Reply:
> You do not need any of this and can just got with the default, but it might be slower..

Ok, so `overlay` driver is just an optional setting, not required. I've been misunderstanding that. Then, +1.